### ISSUE TITLE
serialize() & magic_quotes_gpc problem

### ISSUE THREAD

**Issue by agaertner** [stevenosloan/heroku-buildpack-apache#2]
Do I need to clone this entire repo when I want to disable the magic_quotes_gpc in the php.ini ? It is ruining the serialized objects I send with cURL on the servers end.

**Comment by stevenosloan** [comment]
This is really an old, unmaintained project so I wouldn't suggest using it in production, but the official php buildpack instead. Instructions for modifying php.ini can be found [here](https://devcenter.heroku.com/articles/custom-php-settings#php-runtime-settings).

*But* if you do, you shouldn't need to clone it directly -- you would just set it as the buildpack for your app on heroku [as described here](https://github.com/stevenosloan/heroku-buildpack-apache#use).

**Comment by agaertner** [comment]
I fixed it now. I think it was, because I used base64 encode / serialize from an older php version and encoded / unserialized on php ^7   . I don't know exactly why, but it is working now.

Why I thought it was magic_quotes_gpc was because I researched and someone had the same problem with serialize and they concluded that serialize got corrupted through magic_quotes_gpc on the servers end. It sounded legit to me that this was also my problem.  I really do not know what exactly I did different this time, but it it is working.
